docs/integrations/destinations/clickhouse-migrations.md

```diff
@@ -1,54 +1,86 @@
-# Clickhouse Migration Guide
+# ClickHouse Migration Guide
 
-## SSH Support :warning:
+## SSH Support
 
-SSH is implementation for the new connector is in Beta. If you upgrade and SSH
-does not work for you, please reach out to support.
+:::warning
+SSH tunneling support for the ClickHouse connector is currently in **Beta**.
+:::
 
 ## Upgrading to 2.0.0
 
-This version differs from 1.0.0 radically. Whereas 1.0.0 wrote all your data
-as JSON to raw tables in airbyte_internal database, 2.0.0 will properly separate
-your schema into typed columns and write to the specified database in the
-configuration and the un-prefixed table name. You will no longer see
-`airbyte_internal.{database}_raw__stream_{table}` and will instead see
-`{database}.{table}`.
+Version 2.0.0 represents a fundamental architectural change from version 1.0.0.
 
-While is treated as a "breaking change", connections should continue to function
-with no changes, albeit writing data to a completely different location and in a
-different form. So any downstream pipelines will need updating to ingest the new
-data location / format.
+**Version 1.0.0 behavior:**
 
-## Migrating existing data to the new format
+- Wrote all data as JSON to raw tables in the `airbyte_internal` database
+- Used table names like `airbyte_internal.{database}_raw__stream_{table}`
 
-Unfortunately Airbyte has no way to migrate the existing raw tables to the new
-typed format. The only "out of the box" way to get your data into the new format
-is to re-sync it from scratch.
+**Version 2.0.0 behavior:**
 
-## Removing the old tables
+- Writes data to typed columns matching your source schema
+- Creates tables in the configured database with clean table names: `{database}.{table}`
+- No longer uses the `airbyte_internal` database or `_raw__stream_` prefixes
 
-Because the new destination has no knowledge of the old destination's table
-naming semantics, we will not remove existing data. If you would like to, you
-will need to delete all the tables saved in the old format, which for most
-people should be under `airbyte_internal.{database}_raw__`, but may vary based
-on your specific configuration.
+While this is a breaking change, existing connections continue to function after upgrading. However, data is written to a completely different location in a different format. **You must update any downstream pipelines** (SQL queries, BI dashboards, data transformations) to reference the new table locations and schema structure.
 
-## Gotchas
```

Comment:
I do _not_ like removing the gotchas header. This draws attention to very specific issues encountered by a high number of users and how to fix them. The new format makes these less obvious by nesting them in a broader categorical information architecture.